src/cmd-generate-release-meta

```diff
@@ -146,6 +146,17 @@ def append_build(out, input_):
                     "image": cloud_dict[image_field]
                 }
 
+    # IBMCloud/PowerVS specific additions
+    for meta_key, cloud, object_field, bucket_field, url_field in ("ibmcloud", "ibmcloud", "object", "bucket", "url"), ("powervs", "powervs", "object", "bucket", "url"):
+        if input_.get(meta_key, None) is not None:
+            arch_dict["media"].setdefault(cloud, {}).setdefault("objects", {})
+            for cloud_dict in input_.get(meta_key):
+                arch_dict["media"][cloud]["objects"][cloud_dict["region"]] = {
```

Comment:
```suggestion
                arch_dict["media"][cloud]["images"][cloud_dict["region"]] = {
```

I'm thinking we should probably keep this as `images:` here. I realize that's kind of inconsistent with other things I've told you, but theoretically here we could start creating an actual image (not just the object storage object) in IBM (for ibmcloud VPC gen2 and maybe in the future for powervs) and we'd still use this same structure for that information.

Reply:
yeah i had the same doubt too..yes makes sense as we will use this for the bootimage as well.